stories/screen.js

```diff
@@ -102,7 +103,8 @@ storiesOf('Screens', module)
   .add('Invoice QR', () => (
     <InvoiceQR store={store} invoice={invoice} nav={nav} />
   ))
-  .add('Loader - First Time', () => <Loader />);
+  .add('Loader - First Time', () => <Loader />)
+  .add('Seed Verify', () => <SeedVerify seedCheck={store.seedCheck} />);
```

Comment:
This won't work if `store.seedCheck` changes, as it will reference an old instance. We should inject the `store` here to stay consistent with other views.

Reply:
my bad, forgot to make this change consistently when going through my PR backlog :sweat_smile: